mpqp/execution/connection/azure_connection.py

```diff
@@ -0,0 +1,126 @@
+from getpass import getpass
+from typing import TYPE_CHECKING
+from termcolor import colored
+
+from mpqp.execution.connection.env_manager import get_env_variable, save_env_variable
+
+if TYPE_CHECKING:
+    from azure.quantum.qiskit import AzureQuantumProvider
+    from azure.quantum import Workspace
+
+
+def config_azure_account():
+    """
+    Configure azure account with resource_id and  Location.
+
+    Returns:
+        tuple: A message indicating the result of the configuration and an empty list.
+    """
+    was_configured = get_env_variable(f"AZURE_CONFIGURED") == "True"
+
+    if was_configured:
+        decision = input(
+            f"AZURE account is already configured. Do you want to update it? [y/N]"
+        )
+        if decision.lower().strip() != "y":
+            return "Canceled.", []
+
+    resource_id = getpass(f"Enter your AZURE resource_id: ")
```

Comment:
in the doc (more specifically in getting-started#setup-remote-accesses), explain how to get these values

Reply:
done